### PR TITLE
Update number of integrations

### PR DESCRIPTION
**Pull request by MrBartusek**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ Open source home automation that puts local control and privacy first. Powered b
 
 - [Get started](https://www.home-assistant.io/getting-started/)
 - [View demo](https://demo.home-assistant.io/)
-- [Browse 1900+ integrations](https://www.home-assistant.io/integrations/)
+- [Browse 2100+ integrations](https://www.home-assistant.io/integrations/)
```
Comment (frenck) [inline profile/README.md:9]:
```suggestion
- [Browse 1000+ integrations](https://www.home-assistant.io/integrations/)
```

CleanShot 2022-08-16 at 14 02 49

**Review by frenck — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 1 above)
Comment (MrBartusek) [reply profile/README.md:9]: 👀

image

Comment (frenck) [reply profile/README.md:9]: Yup, we will change that too (soon™). The problem is that the website counts platforms in some cases (old school).

Comment (MrBartusek) [reply profile/README.md:9]: Okay but why not 1800+ then?

Comment (frenck) [reply profile/README.md:9]: I don't get that comment? Why would 1800 make sense?

Comment (MrBartusek) [reply profile/README.md:9]: Honestly i thought that this 0 is actually 8 since low photo quality 😅 sorry for that

Comment (frenck) [reply profile/README.md]: hehe lol OK, understand 👍

**Review by frenck — APPROVED** [review]
Thanks, @MrBartusek 👍

**Pull request merged**